src/swap/hooks/useSwapBalances.tsx

```diff
@@ -37,12 +46,21 @@ export function useSwapBalances({
   const toTokenBalanceError = isToNativeToken
     ? ethBalanceError
     : toBalanceError;
+  const fromTokenResponse = isFromNativeToken
+    ? ethBalanceResponse
+    : _fromTokenResponse;
+  const toTokenResponse = isToNativeToken
+    ? ethBalanceResponse
+    : _toTokenResponse;
 
   return useValue({
     fromBalanceString,
     fromTokenBalanceError,
 
     toBalanceString,
     toTokenBalanceError,
+
```

Comment:
Why are we adding those places, if you want to add spaces to split the meaning, we probably just want to add a line of comments between them, and explain more clearly the three groups.

Reply:
added to the two previous groups